### ISSUE TITLE
ws tab should have `ws://` pre-text at urlbar.

### ISSUE THREAD

**Issue by Nishchit14** [firecamp-io/Firecamp#161]
The ws tab should have `ws://` pre-text at urlbar for better accessibility and understanding. 
![image](https://user-images.githubusercontent.com/5078921/122763386-e5bcb900-d2bb-11eb-9b69-b15427e5dd1c.png)

**Comment by Nishchit14** [comment]
released https://github.com/firecamp-io/firecamp/releases/tag/v2.0.11
